pandas/tests/base/test_conversion.py

```diff
@@ -271,7 +271,7 @@ def test_numpy_array_all_dtypes(any_numpy_dtype):
         (pd.core.arrays.period_array(["2000", "2001"], freq="D"), "_data"),
         (pd.core.arrays.integer_array([0, np.nan]), "_data"),
         (pd.core.arrays.IntervalArray.from_breaks([0, 1]), "_left"),
```

Comment:
can you change the others to be pd.arrays here

Reply:
also / maybe better, just import things at the top rather than using pd.*

Reply:
`IntervalArray` is OK.  `period_array` and `integer_array` need to stay because of the type conversions done by those initializers.  I left those in as they are rather than import them.